src/olympia/devhub/templates/devhub/addons/submit/source.html

```diff
@@ -0,0 +1,91 @@
+{% extends "devhub/addons/submit/base.html" %}
+
+{% block title %}{{ dev_page_title(_('Source Code Upload'), addon) }}{% endblock %}
+
+{% block primary %}
+
+{% set channel_param = 'listed' if channel == amo.RELEASE_CHANNEL_LISTED else 'unlisted' %}
+
+<form method="post" enctype="multipart/form-data" id="submit-source">
+    {% csrf_token %}
+    <h3>{{ _('Submit Source Code') }}</h3>
+    <div>
+        {{ form.non_field_errors() }}
+        <p>
+        {% trans %}
+            You may need to submit source code. If you don’t submit the correct
+            source code now, your add-on may be removed from the site!
+        {% endtrans %}
+        </p>
+        <p id="source-code-submission-question">
+        {% trans a_attrs = 'href="https://developer.mozilla.org/en-US/Add-ons/Source_Code_Submission" target="_blank" rel="noopener noreferrer"'|safe %}
+            Please review the <a {{ a_attrs }}>source code submission policy</a>. Do you use any of the following?
+        {% endtrans %}
+        </p>
+        <ul id="source-code-example">
+            <li>{% trans %}Code minifiers or obfuscators{% endtrans %}</li>
+            <li>
+            {% trans a_attrs = 'href="https://webpack.js.org/" target="_blank" rel="noopener noreferrer"'|safe %}
+                Tools that combine multiple files into a single file, such as <a {{ a_attrs }}>webpack</a>
```

Comment:
Uhm, I am not sure why, but it seems the PRD missed the latest comment there. @jvillalobos had requested that link to be removed. Can we just make "webpack" italic instead, please?

Sorry for that.

Reply:
umm, it's not italic in the PRD.

Reply:
https://github.com/mozilla/addons-server/pull/9086/commits/b23efc2fe2f9abf30066d046c14dd55a7d7e12ee